packages/bits/src/lib/wizard-v2/wizard-vars.less

```diff
@@ -3,14 +3,15 @@
 @wizard-header-max-height: @wizard-header-height;
 
 @wizard-label-header-height: 24px;
+@wizard-label-header-width: 145px;
 @wizard-label-position-bottom-top-gap: 16px;
 @wizard-label-min-width: 50px;
 
 @wizard-vertical-content-margin: 36px;
 
-@wizard-side-gap: 24px;
-@wizard-line-width: 1px;
+@wizard-side-gap: 15px;
+@wizard-line-width: 2px;
 @wizard-line-gap: 0px;
 
 @wizard-step-sub-label-font-size: 12px;
-@wizard-step-header-icon-size: 12px;
+@wizard-step-header-icon-size: 16px;
```

Comment:
nui-framework-icon-sizes.less already has a variable for this: @icon-size-default